t/mojo/prefork.t

```diff
@@ -76,6 +141,13 @@ $prefork->app->hook(
   before_server_start => sub {
     my ($server, $app) = @_;
     push @server, $server->workers, $app->mode;
+    Mojo::UserAgent->new->post("$durl/server/$$");
+  }
+);
+$prefork->app->hook(
+  before_app_start => sub {
+    my ($app) = @_;
```

Comment:
It's not a list.

Reply:
Aren't parameters always passed as a LIST in Perl?  Even if there is a single one?

Reply:
Yes, but we use the convention `my $app = shift` for single arguments. It became an idiom because it used to be a little faster with some versions of Perl (i don't actually remember the details, ask on IRC if you want to know more).

Reply:
Yeah, I remember that discussion on p5p at the time.  It was a loooong time ago.  In my own code, I just don't care and prefer to say `($x) = @_;` even if there is just one argument because I favor readability over hypothetical performance.  My only exception is for `my $self = shift;` which I always use on a line by itself, even if there are following arguments.

But `Mojolicious` is not _my_ code and I will follow your standards by amending this code once again.

Reply:
If it was up to me alone we would just be using signatures everywhere. 😉